pagopa/io-app
# chore: [IOBP-368] Revamp wallet transaction detail page

## Short description
This PR integrates the revamp of the new wallet transaction detail page

## List of changes proposed in this pull request
- Added into `walletV3` feature folder structure the `transaction` folder
- Added the redux flow and saga mock (that doesn't make a real API request until we switch from actual PM to BIZ Event);
- Added two new screens: `WalletTransactionDetailsScreen` and `WalletTransactionOperationDetailsScreen`
- Added Skeletons showing up while loading some API requests;
- Connected redux flow & sagas to the `walletV3` feature store;
- Added translation locales;
- Edited the current `navigateToTransactionDetailsScreen` redirecting to the new screen;

## How to test
- Go into the transactions list from the wallet home;
- Select any transaction from the list;
- You should be able to see the new transaction screen;

## Revamp design
https://www.figma.com/file/SDaOSTsQnJUxmoPQEfz3bJ/Ricevuta-di-pagamento-pagoPA?type=design&node-id=520-10951&mode=design&t=KSENC2iM2XXGMJaT-4

## Preview
|Before|After|
|-|-|
| <video src="https://github.com/pagopa/io-app/assets/34343582/a9b8d7c6-4e1b-408a-a4bd-3d64c0d36cf2" />  |<video src="https://github.com/pagopa/io-app/assets/34343582/cf17a14f-ed57-4796-93b5-dcf4c8d71b07" />

**NB:** In the new transaction preview video there is a 2-second forced loading to show up skeletons to showcase how the loading page looks when it is loading
